src/provider.tsx

```diff
@@ -36,24 +36,43 @@ export const IntercomProvider: React.FC<IntercomProviderProps> = ({
       ].join(''),
     );
 
-  if (!isSSR && !window.Intercom && shouldInitialize) {
+  const boot = React.useCallback(
+    (props?: IntercomProps) => {
+      if (!window.Intercom && !shouldInitialize) {
+        logger.log(
+          'warn',
+          'Intercom instance is not initialized because `shouldInitialize` is set to `false` in `IntercomProvider`',
+        );
+        return;
+      }
+
+      if (isBooted.current) return;
+
+      const metaData = {
```

Comment:
Can you add the `RawIntercomBootProps` back in?

Reply:
:+1: